components/global/product_switcher.tsx

```diff
@@ -107,56 +135,55 @@ const ProductSwitcher = (): JSX.Element => {
 
     return (
         <div ref={menuRef}>
-            <IconButton
-                icon={'products'}
-                onClick={handleClick}
-                size={'sm'}
-                toggled={switcherOpen}
-                inverted={true}
-            />
-            {showTutorialStep && <ProductSwitcherTip/>}
-            <SwitcherMenu
+            <MenuWrapper
                 open={switcherOpen}
             >
-                <SwitcherMenuDescriptiveText>
-                    <FormattedMessage
-                        defaultMessage='Open...'
-                        id='global_header.open'
+                <ProductSwitcherContainer>
+                    <IconButton
+                        icon={'products'}
+                        onClick={handleClick}
+                        size={'sm'}
+                        compact={true}
+                        toggled={switcherOpen}
+                        inverted={true}
+                        aria-label='Select to open product switch menu.'
                     />
-                </SwitcherMenuDescriptiveText>
-                <SwitcherNavEntry
-                    destination={'/'}
-                    icon={<ChannelsIcon/>}
-                    text={'Channels'}
-                    active={currentProductID === null}
-                />
-                {items}
-            </SwitcherMenu>
+                    <ProductBranding>
```

Comment:
This needs to be resolved. My suggestion is to remove it for now. 
It's also a bit confusing to be here anyway. It probably should be with the other product specific stuff or at the top level of the global header component? It's a bit confusing to have it lumped with the switcher menu.

Reply:
That one's on me, I did the initial setup and forgot I hadn't updated to dynamically adjust based on product.